From 4a1590d5a99f1077631f407a66e29c1f611a7250 Mon Sep 17 00:00:00 2001
From: Mark Vis <118430440+mark-vis@users.noreply.github.com>
Date: Tue, 6 Oct 2026 01:03:51 +0200
Subject: [PATCH 17/32] (b) v2, optional O1: .tlg of firstaid-cleveref-1684

O1 stores the list text of a caption in \@currentlabelname, so the
\newlabel written by the first-aid test for cleveref now has the title
"Simple Table" in its third field. The firstaid tests were not run when
O1 was added; this updates the expected output (one line, all engines).
---
 required/firstaid/testfiles/firstaid-cleveref-1684.tlg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/required/firstaid/testfiles/firstaid-cleveref-1684.tlg b/required/firstaid/testfiles/firstaid-cleveref-1684.tlg
index e14470789..34ae3b506 100644
--- a/required/firstaid/testfiles/firstaid-cleveref-1684.tlg
+++ b/required/firstaid/testfiles/firstaid-cleveref-1684.tlg
@@ -70,7 +70,7 @@ Completed box being shipped out [1]
 .......\OT1/cmr/m/n/10 e
 ......\glue 0.0 plus 1.0fil
 .....\glue 0.0
-.....\write1{\newlabel{tab:Simple Table}{{1}{\thepage }{}{table.1}{}}}
+.....\write1{\newlabel{tab:Simple Table}{{1}{\thepage }{Simple Table}{table.1}{}}}
 .....\write1{\newlabel{tab:Simple Table@cref}{{[table][1][]1}{[1][1][]\thepage }{}{}{}}}
 .....\glue 0.0
 ...\glue 12.0 plus 2.0 minus 2.0
